### PR TITLE
Update id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel

### PR DESCRIPTION
**Pull request by bwaldon**
"The Secretary shall establish an advisory committee..."

### CODE REVIEW

File: datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel [hunk 1]
```diff
@@ -1,4 +1,71 @@
...
+                                                                :Mod (PP
+                                                                    :Head (P :t "in")
+                                                                    :Obj (NP
+                                                                        :Det (DP
+                                                                            :Head (D :t "this"))
+                                                                        :Head (Nom
+                                                                            :Head (N :t "section"))))
+                                                                :Head (VP
+                                                                    :Head (V :t "referred" :l "refer" :xpos "VBN")
+                                                                    :Particle (PP
```
Comment (nschneid) [inline datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel:63]: Comp

File: datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel [hunk 2]
```diff
@@ -1,4 +1,71 @@
...
+                                            :Head (V :t "known" :l "know" :xpos "VBN")
+                                            :Comp (PP
+                                                :Head (P :t "as")
+                                                :PredComp (NP
+                                                    :Det (DP
+                                                        :Head (D :t "the"))
+                                                    :Head (Nom
+                                                        :Head (Nom
+                                                            :Mod (Nom
+                                                                :Head (N :t "Advisory"))
```
Comment (nschneid) [inline datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel:40]: "advisory" above is an Adj (which I think makes more sense than N unless it is "an advisory")

File: datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel [hunk 3]
```diff
@@ -1,4 +1,71 @@
...
+                :Head (V :t "establish" :xpos "VB")
+                :Obj (NP
+                    :Det (DP
+                        :Head (D :t "an"))
+                    :Head (Nom
+                        :Head (Nom
+                            :Mod (AdjP
+                                :Head (Adj :t "advisory"))
+                            :Head (N :t "committee"))
+                        :Mod (Clause
```
Comment (nschneid) [inline datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel:24]: supplement maybe? it doesn't feel restrictive. but I'm not sure

File: datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel [hunk 4]
```diff
@@ -1,4 +1,71 @@
...
+                                                                    :Head (V :t "referred" :l "refer" :xpos "VBN")
+                                                                    :Particle (PP
+                                                                        :Head (P :t "to"))
+                                                                    :Comp (PP
+                                                                        :Head (P :t "as")
+                                                                        :PredComp (NP
+                                                                            :Det (DP
+                                                                                :Head (D :p "``" :t "the"))
+                                                                            :Head (Nom
+                                                                                :Head (N :t "Committee" :p "''" :p ")" :p ".")))))))))))))))))))))
```
Comment (nschneid) [inline datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel:71]: I see there has been normalization of quotes relative to the input string, not sure if that was intentional

**Review by nschneid — CHANGES_REQUESTED** [review]

File: datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel (hunk 3 above)
Comment (bwaldon) [reply datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel:24]: I agree — will update

**Pull request merged**